docs/reference/analysis/tokenfilters/synonyms-format.asciidoc

```diff
@@ -15,7 +15,7 @@ This format uses two different definitions:
 ipod, i-pod, i pod
 computer, pc, laptop
 ----
-* Explicit mappings: Matches a group of words to other words. Words on the left hand side of the rule definition are expanded into all the possibilities described on the right hand side. Example:
+* Explicit synonyms: Matches a group of words to other words. Words on the left hand side of the rule definition are expanded into all the possibilities described on the right hand side. Example:
```

Comment:
I liked the previous wording "mappings" better, as this is not explicit synonyms, but explicit mapping between synonyms.  Or we can also say "Explicit synonym rules"

Reply:
If possible, I would like to avoid the word "mappings" because it could be confused with index mappings. "Explicit synonym rules" and "Equivalent synonym rules" sounds good to me though.

Reply:
I like synonym rules better - we've been using that wording for synonyms API and does not contain the mapping ambiguity. Changing it!